Blockzilla/BrowserViewController.swift

```diff
@@ -537,20 +535,36 @@ class BrowserViewController: UIViewController {
         SKStoreReviewController.requestReview()
     }
 
-    fileprivate func showSettings() {
+    fileprivate func showSettings(shouldScrollToSiri: Bool = false) {
         guard let modalDelegate = modalDelegate else { return }
         
         urlBar.shouldPresent = false
         
-        let settingsViewController = SettingsViewController(searchEngineManager: searchEngineManager, whatsNew: browserToolbar.toolset)
-        
+        let settingsViewController: UIViewController!
+        if shouldScrollToSiri {
+            settingsViewController = SettingsViewController(searchEngineManager: searchEngineManager, whatsNew: browserToolbar.toolset, shouldScrollToSiri: true)
+        } else {
+           settingsViewController = SettingsViewController(searchEngineManager: searchEngineManager, whatsNew: browserToolbar.toolset)
+        }
         let settingsNavController = UINavigationController(rootViewController: settingsViewController)
         settingsNavController.modalPresentationStyle = .formSheet
         
         modalDelegate.presentModal(viewController: settingsNavController, animated: true)
 
         Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.click, object: TelemetryEventObject.settingsButton)
     }
+    
+    @available(iOS 12.0, *)
```

Comment:
I feel like there is a lot of `@available(iOS 12.0, *)` code in here for the Siri tips. Could it be simplified to a static property or function on TipManager? Should make cleanup a bit easier when this is no longer needed.

 `static func showSiriTips() { ...@avaialble stuff here ` ?

Reply:
Cleaned up significantly, though still checking once in TipManager and once in bvc to handle displaying SiriSettings in bvc.